parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetMemoryManagerRuntimeException.java

```diff
@@ -0,0 +1,38 @@
+/* 
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.hadoop;
+
+import org.apache.parquet.ParquetRuntimeException;
+
+/**
+ * Signals that an exception of some sort has occurred with the Parquet Memory
+ * Manager.
+ *
+ * @see MemoryManager
+ */
+public class ParquetMemoryManagerRuntimeException
+    extends ParquetRuntimeException {
+
+  private static final long serialVersionUID = 1L;
+
+  public ParquetMemoryManagerRuntimeException(String format) {
```

Comment:
The argument naming is a bit misleading. `message` or similar would be a better choice.

Reply:
Good catch. Thanks. Updated.